board.rb

```diff
@@ -0,0 +1,90 @@
+class Board
+
+    def initialize(player_one, player_two)
+      @player_one = player_one
+      @player_two = player_two
+      if @player_one.get_index == 1
+        @current_player = @player_one
+      else
+        @current_player = @player_two
+      end
```

Comment:
Can be changed to `@current_player = @player_one.get_index == 1 ? @player_one : @player_two`